File: Exercicios_PYTHON/Exercicio_009.py
import pandas as pd

while True:
    try:
        Dia_Nasc = int(input("Digite o dia que você nasceu: "))
        Mes_Nasc = int(input("Digite o mês que você nasceu: "))
        Ano_Nasc = int(input("Digite o ano que você nasceu: "))
        break
    except ValueError:
        print("Erro!! Digite apenas números!!")

Data_AtualGet = pd.to_datetime('today')

Dia = Data_AtualGet.day
Mes = Data_AtualGet.month
Ano = Data_AtualGet.year

if (Mes, Dia) >= (Mes_Nasc, Dia_Nasc):
    idade = Ano - Ano_Nasc
else:
    idade = Ano - Ano_Nasc - 1

print("Você tem", idade, "anos")

File: Exercicios_JAVA/Exercicio_009.java
import java.util.Scanner;
import java.time.*;

public class Exercicio_009 {
    /*
     * EXERCICIO 9: Faça um algoritmo que calcule a idade de uma pessoa a partir do
     * dia, mes e ano em que ela nasceu
     */
    public static void main(String[] args) {
        try (Scanner input = new Scanner(System.in)) {
            System.out.println("Digite o dia em que voce nasceu: ");
            int Dia_nasc = input.nextInt();

            System.out.println("Digite o mes em que voce nasceu: ");
            int Mes_nasc = input.nextInt();

            System.out.println("Digite o ano em que voce nasceu: ");
            int Ano_nasc = input.nextInt();

            int Dia_atual = LocalDate.now().getDayOfMonth();
            int Mes_atual = YearMonth.now().getMonthValue();
            int Ano_atual = Year.now().getValue();

            if (Dia_atual >= Dia_nasc && Mes_atual >= Mes_nasc) {
                int idade = Ano_atual - Ano_nasc;
                System.out.println("Voce tem " + idade + " anos");
               
            }

            else {
                int Idade = Ano_atual - Ano_nasc - 1;
                System.out.println("Voce tem " + Idade + " anos ");
                
            }

        }

    }
}
